app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java

```diff
@@ -50,18 +70,29 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
         View view = getView();
         scheduleRecyclerView = (RecyclerView) view.findViewById(R.id.schedule_list);
-        getScheduleEvents(API.nmaService);
-
     }
 
     private void getScheduleEvents(final NMAService nmaService) {
         nmaService.getEvents().enqueue(new Callback<List<ScheduleEvent>>() {
             @Override
             public void onResponse(Call<List<ScheduleEvent>> call,
                                    Response<List<ScheduleEvent>> response) {
-                List<ScheduleEvent> scheduleEvents = response.body();
-                if (getContext() != null) {
-                    setScheduleItems(scheduleEvents);
+                Context context = getContext();
+                if (context != null) {
+                    List<ScheduleEvent> scheduleEvents = response.body();
+                    SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context.getApplicationContext());
+                    int sectionId = Integer.parseInt(sharedPreferences.getString(getString(R.string.section_key), "0"));
```

Comment:
`sharedPreferences` has method `getInt`. Of course make sure that you saved `int`, not `string`